src/nci_adapter.c

```diff
@@ -366,8 +366,11 @@ nci_adapter_presence_check_timer(
 {
     NciAdapter* self = THIS(user_data);
     NciAdapterPriv* priv = self->priv;
+    NfcTargetSequence* seq = self->target->sequence;
+    gboolean do_presence_check = !seq || (nfc_target_sequence_get_flags(seq)
```

Comment:
```
src/nci_adapter.c: In function 'nci_adapter_presence_check_timer':
src/nci_adapter.c:370:43: warning: implicit declaration of function 'nfc_target_sequence_get_flags'; did you mean 'nfc_target_sequence_flags'? [-Wimplicit-function-declaration]
     gboolean do_presence_check = !seq || (nfc_target_sequence_get_flags(seq)
                                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                           nfc_target_sequence_flags
```
Sorry, it's been renamed it :/

Reply:
I have fixed that and rebased on top of current master. Thanks